scripts/install/config.default.sh

```diff
@@ -41,3 +42,5 @@ SSR_PORT="4100"
 SSR_PWA_PORT=
 
 ADD_B2B_LIBS=false
+
+ADD_PRODUCT_CONFIGURATOR=false
```

Comment:
AFAIK Product configurator feature is not conflicting with any other existing features. So I'm in favour of enabling it by default (set to `true`).

FYI We introduced b2b=false, because it changes in a lot of ways how Storefront works at the moment. In the near future we plan to deploy one storefront, and drive it b2b/b2c in runtime (depending on the `channel` assigned for a `baseSite` in CMS). https://github.com/SAP/spartacus/issues/10251

Reply:
ok, will change that